stravalib/model.py

```diff
@@ -149,58 +194,128 @@ def naive_datetime(value: Optional[AllDateTypes]) -> Optional[datetime]:
 class DeprecatedSerializableMixin(BaseModel):
     """
     Provides backward compatibility with legacy BaseEntity
+
+    Inherits from the `pydantic.BaseModel` class.
     """
 
+    # TODO: i used the output type from this class
+    # could dict also be an int or float??
     @classmethod
-    def deserialize(cls, attribute_value_mapping: Dict):
+    def deserialize(cls, attribute_value_mapping: dict[str, str]) -> Self:
```

Comment:
I think the best type for the dict would be `dict[str, Any]`

Reply:
Fixed.

Are you suggesting Any because we don't know what will be in the attribute value mapping dict? I wasn't sure how to handle these instances where i wasn't sure what the range of possible values was but was nervous about just using `Any`

Reply:
Exactly, we have no idea what will come in the dict. Could even be more nested dict. So Any is the more appropriate type

Reply:
ok got it. thanks! i'll resolve this now!